pkgs/applications/misc/cardpeek/default.nix

```diff
@@ -29,33 +28,30 @@ stdenv.mkDerivation {
   postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
     # replace xcode check and hard-coded PCSC framework path
     substituteInPlace configure.ac \
-      --replace 'if test ! -e "/Applications/Xcode.app/"; then' 'if test yes != yes; then' \
-      --replace 'PCSC_HEADERS=`ls -d /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*.sdk/System/Library/Frameworks/PCSC.framework/Versions/Current/Headers/ | sort | head -1`' 'PCSC_HEADERS=${PCSC}/Library/Frameworks/PCSC.framework/Headers'
+      --replace-fail 'if test ! -e "/Applications/Xcode.app/"; then' 'if test yes != yes; then' \
+      --replace-fail 'PCSC_HEADERS=`ls -d /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*.sdk/System/Library/Frameworks/PCSC.framework/Versions/Current/Headers/ | sort | head -1`' 'PCSC_HEADERS=$SDKROOT/System/Library/Frameworks/PCSC.framework/Versions/Current/Headers'
```

Comment:
FWIW, framework headers are always available if you pass `-framework`. This is fine, but a better patch would be to drop the `PCSC_HEADERS` line and replace the `-I` with `-framework PCSC`. What it’s currently doing is not idiomatic for the macOS toolchain.